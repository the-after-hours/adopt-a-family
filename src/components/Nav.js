import React, { Component } from 'react';
import { Link } from 'react-router-dom';

class Nav extends Component {

  render () {
    return (
      <div>
        <div className='header-text'>
          <Link to='/'>
            Adopt A Family
          </Link>
        </div>
        <div className="header-links">
          <div className="header-mission">
            Mission
          </div>
          <div className="header-family">
            Our Families
          </div>
          <div className="header-about">
            <Link to="/aboutus">
              About us
            </Link>
          </div>
          <div className="header-donate">
            Donate
          </div>
          <Link to="/registration">
            <button className="button-login">
              ADOPT A FAMILY
            </button>
          </Link>
        </div>
      </div>
    );
  }
}

export default Nav;
