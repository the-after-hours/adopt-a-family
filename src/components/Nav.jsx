import React, { Component } from 'react';
import { Link } from 'react-router-dom';

class Nav extends Component {
  render() {
    return (
      <div className="header-container">
        <div className="col-3 header-group header-text">
          <Link to="/">Adopt A Family</Link>
        </div>

        <div className="col-8 header-group header-links-group">
          <Link className="header-links" to="/mission">
            Mission
          </Link>
          <Link className="header-links" to="/families">
            Our Families
          </Link>
          <Link className="header-links" to="/aboutus">
            About us
          </Link>
          <Link className="header-links" to="/donate">
            Donate
          </Link>
          <Link className="header-links" to="/registration">
            <button className="btn-registration">Adopt A Family</button>
          </Link>
        </div>
      </div>
    );
  }
}

export default Nav;
