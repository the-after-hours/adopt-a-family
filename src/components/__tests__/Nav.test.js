import React from 'react';
import Nav from '../Nav';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import customTheme from '../../theme';

const context = {
  muiTheme: customTheme
};

describe('Main', () => {
  test('Main snapshot', () => {
    const component = shallow(<Nav />, { context });
    const tree = toJson(component);

    expect(tree).toMatchSnapshot();
  });
});