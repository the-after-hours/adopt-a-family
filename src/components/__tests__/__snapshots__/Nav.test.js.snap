// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Main snapshot 1`] = `
<div>
  <div
    className="header-text"
  >
    <Link
      replace={false}
      to="/"
    >
      Adopt A Family
    </Link>
  </div>
  <div
    className="header-links"
  >
    <div
      className="header-mission"
    >
      Mission
    </div>
    <div
      className="header-family"
    >
      Our Families
    </div>
    <div
      className="header-about"
    >
      <Link
        replace={false}
        to="/aboutus"
      >
        About us
      </Link>
    </div>
    <div
      className="header-donate"
    >
      Donate
    </div>
    <Link
      replace={false}
      to="/registration"
    >
      <button
        className="button-login"
      >
        ADOPT A FAMILY
      </button>
    </Link>
  </div>
</div>
`;
